tests/resources/dsl/without_policies.yaml

```diff
@@ -0,0 +1,26 @@
+# this blueprint is like with_policies1, but without the policies
+# - so that we have two simply blueprints that only differ by the policies
```

Comment:
thanks!
CI somehow managed to run an older version of this branch - the failing assert isn't even in the code, lol. Probably because of the merge conflict. I'll rebase it tomorrow.